pkg/corerp/renderers/gateway/render.go

```diff
@@ -37,6 +37,9 @@ import (
 	resources_kubernetes "github.com/radius-project/radius/pkg/ucp/resources/kubernetes"
 )
 
+const ErrSecretStoreNotFound = "secretStore resource %s not found"
```

Comment:
nit: does this need to be exported from the package? looks like it's only used by the unit tests.

Also, I think it would be better to avoid the prefix `Err`. Usually that's used when a variable is a static `error` value, in this case it's a format string.